man/common/authorizations.md

```diff
@@ -9,17 +9,21 @@ Authorization for use of an object in TPM2.0 can come in 3 different forms:
 
 ## Passwords
 
-Passwords are interpreted in two forms, string and hex-string. A string password is not
+Passwords are interpreted in three forms; string, hex-string and file. A string password is not
```

Comment:
shouldn't this be and **a** file ?

Reply:
Well spotted, I think this should probably be "string, hex-string or a file". Will address.